css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-content-right {
+		display: flex;
+		flex-grow: 1;
+		padding-block: $hero-padding-xs;
+
+		@include tablet {
+			justify-content: flex-end;
+			padding-block: $hero-padding-sm;
+			padding-inline-start: $hero-padding-xs;
+		}
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-start: $hero-padding-sm;
+			padding-inline-end: $hero-padding-sm;
+		}
+
+		.hero-right-card {
+			align-self: flex-start;
+			background: #ffffffd9;
+			color: $black-static;
+			box-shadow: 0 0 50px 0 #17171726;
+			backdrop-filter: blur(20px);
+
+			.theme-high-contrast & {
+				background: #ffffffd9;
+			}
+
+			@include desktop {
+				background: linear-gradient(116deg, #ffffffd9 13.94%, #ffffff59 111.37%);
+				margin-inline-start: $hero-padding-xs;
+			}
+		}
+	}
+
+	&.hero-right-content {
```

Comment:
name mismatch?

Reply:
part of the above smell.  Wanting to make sure we're separating concern on heros with right content.  So naming will get cleaned up today.